specification/support/resource-manager/Microsoft.Support/preview/2019-05-01-preview/support.json

```diff
@@ -1406,22 +1414,22 @@
             }
         },
         "QuotaChangeRequest": {
-            "description": "Quota change request information.",
+            "description": "This property is required for providing the region and new quota limits",
             "type": "object",
             "properties": {
                 "region": {
-                    "description": "Region for which the quota change request is being made.",
+                    "description": "Region for which the quota increase request is being made.",
                     "type": "string"
                 },
                 "payload": {
-                    "description": "The payload of the quota change request.",
+                    "description": "Payload of the quota increase request.",
                     "type": "string"
                 }
             }
 
         },
         "UpdateSupportTicket": {
-            "description": "Updates support ticket.",
+            "description": "Ability to update severity and contact details in the support ticket.",
```

Comment:
nit: **Ability** to update ...  feels like describing a boolean flag.  How about: "**Updates to a support ticket**"

Reply:
Fixed